api/books/booklist.ts

```diff
@@ -1,9 +1,9 @@
 import axios from "axios"
 
-interface GetBooksProps {
+interface GetBookListProps {
```

Comment:
If this is `identical` with the others, better use a single defined `interface` to use. Also, I think we understand this more as a `token` than `auth`.

```
# in another file
export interface TokenProps {
 token: string
}

export getBookListProps extends TokenProps {} # Usually matter of preference, 
# I prefer this, but a lot more others prefer just using TokenProps since there's no change.

export const getBookList = async({ token }: TokenProps) => {
...

export const getDiscover = async({ token }: TokenProps) => {
```

Reply:
good catch! made changes to implement this